xds/src/main/java/io/grpc/xds/EdsLoadBalancer.java

```diff
@@ -312,6 +312,7 @@ public void shutdown() {
           xdsClient.cancelClientStatsReport();
           isReportingLoad = false;
         }
+        localityStore.reset();
```

Comment:
Just for reverting back to the original implementation. Please see changes from all commits.